### PR TITLE
doc: add create server exemple

### PR DESCRIPTION
**Pull request by jerome-quere**

### CODE REVIEW

File: example_test.go [hunk 1]
```diff
@@ -84,6 +87,68 @@ func Example_listServers() {
 	fmt.Println(response)
 }
 
+func Example_createServer() {
```
Comment (jerome-quere) [inline example_test.go]: 60+ line to create a server 😞

File: example_test.go [hunk 2]
```diff
@@ -84,6 +87,68 @@ func Example_listServers() {
...
+	createIPRes, err := instanceAPI.CreateIP(&instance.CreateIPRequest{})
+	if err != nil {
+		panic(err)
+	}
+
+	// Create a new DEV1-S server
+	createRes, err := instanceAPI.CreateServer(&instance.CreateServerRequest{
+		Name:              "my-server-01",
+		CommercialType:    "DEV1-S",
+		Image:             imageID,
```
Comment (jerome-quere) [inline example_test.go]: Would be nice if I could have write `ubuntu-bionic` here

File: example_test.go [hunk 3]
```diff
@@ -84,6 +87,68 @@ func Example_listServers() {
...
+		DynamicIPRequired: scw.BoolPtr(false),
```
Comment (jerome-quere) [inline example_test.go]: Sad that this line is required if you want to use a dynamic IP.

File: example_test.go (hunk 1 above)
Comment (jerome-quere) [reply example_test.go]: We should consider a `Create(Simple|Basic)Server` function

File: example_test.go [hunk 4]
```diff
@@ -84,6 +87,68 @@ func Example_listServers() {
...
+	})
+	if err != nil {
+		panic(err)
+	}
+
+	// Start the server and wait until it's ready.
+	err = instanceAPI.ServerActionAndWait(&instance.ServerActionAndWaitRequest{
+		ServerID: createRes.Server.ID,
+		Action:   instance.ServerActionPoweron,
+		Timeout:  3 * time.Minute,
```
Comment (kindermoumoute) [inline example_test.go]: 10 minutes

File: example_test.go [hunk 5]
```diff
@@ -84,6 +87,68 @@ func Example_listServers() {
...
+	}
+
+	// Create SDK objects for Scaleway Instance and marketplace
+	instanceAPI := instance.NewAPI(client)
+	marketplaceAPI := marketplace.NewAPI(client)
+
+	// Get imageID for ubuntu-bionic from marketplace API
+	imageID, err := marketplaceAPI.GetLocalImageIDByLabel(&marketplace.GetLocalImageIDByLabelRequest{
+		ImageLabel:     "ubuntu-bionic",
+		CommercialType: "DEV1-S",
```
Comment (kindermoumoute) [inline example_test.go]: nit: put `"DEV1-S"` and `"ubuntu-bionic"` in a const at the top

**Pull request merged**